package.json

```diff
@@ -70,6 +70,6 @@
     "tailwindcss": "^2.2.4",
     "ts-node": "^10.0.0",
     "twin.macro": "^2.5.0",
-    "typescript": "4.3.2"
+    "typescript": "^4.5.0-dev.20211030"
```

Comment:
Weird bug vscode auto-import https://github.com/Andrewnt219/rent-near-me/pull/79/commits/282b2681928269094b636f2e25b58409406459b8

Reply:
Yea, I notice the bug as well.

Is this fix the one breaking deployment? I don't see any reason why deployment is breaking besdies this

Reply:
No, the deployment is breaking because other components are using unknown tailwind tokens. I upgraded TS with the previous implementation and it was fine.

Reply:
I see... Good luck updating those